connect/templates/operator-deployment.yaml

```diff
@@ -25,6 +25,12 @@ spec:
         - name: {{ .Values.connect.applicationName }}
           image: {{ .Values.operator.imageRepository }}:{{ .Values.operator.version | default "latest" }}
           imagePullPolicy: {{ .Values.connect.imagePullPolicy }}
+          securityContext:
+            runAsUser: 65532
+            runAsGroup: 65532
+            allowPrivilegeEscalation: false
+            capabilities:
+              add: []
```

Comment:
I am not sure if this does what you expect it to do. Did you mean this:
```yaml
capabilities:
  drop:
    - all
  add: []
```

However, that may revoke more capabilities then we intent to.

Reply:
I was looking for the magic "Give me no special privileges" and missed the `drop` directive.

Reply:
I have verified that with 
```
drop:
  - all
```
the operator continues to work